modules/client/src/main/java/org/apache/ignite/internal/client/ReliableChannel.java

```diff
@@ -130,6 +130,9 @@ public final class ReliableChannel implements AutoCloseable {
     @Nullable
     private ScheduledExecutorService streamerFlushExecutor;
 
+    /** Inflights. */
+    private final ClientTransactionInflights inflights;
```

Comment:
This is global across all server connections. Does it mean that inflight notifications can be received on a connection that is different from the tx coordinator?

I don't see how it is possible in `ClientInboundMessageHandler` - looks like we handle everything within one connection.

Reply:
Inflight tracker is shared between connections, because same directly mapped transaction can be mapped to multiple nodes using different connections.
This is the whole point of direct mapping optimization.